File: Python/160A. Twins.py
def main():
    n = int(input())
    a = list(map(int, input().split()))
    sum1 =0
    for i in range(n):
        sum1 += a[i]
    sum1 //= 2
    sum2, count = 0, 0
    a.sort(reverse=True)
    for i in range(n):
        sum2 += a[i]
        count += 1
        if(sum2>sum1):
            break
    print(count)
main()

File: C++/160A. Twins.cpp
#include<bits/stdc++.h>
using namespace std;
#define FAST_IO ios::sync_with_stdio(false); cin.tie(0); cout.tie(0);
#define endl "\n"
int main()
{
    FAST_IO;
    int n;
    cin>>n;
    vector<int> a;
    int sum1=0;
    for(int i=0; i<n; i++){
        int tmp;
        cin>>tmp;
        sum1 += tmp;
        a.push_back(tmp);
    }
    sum1 /= 2;
    sort(a.begin(), a.end());
    int sum2=0, count=0;
    for(int i=n-1; i>=0; i--){
        sum2 += a[i];
        count += 1;
        if(sum1<sum2){
            break;
        }
    }
    cout<<count<<endl;
    return 0;
}